main.c

```diff
@@ -99,7 +99,7 @@ main(int argc, char **argv)
             exit(10);
         }
 
-        struct job list = {.prev=NULL, .next=NULL};
+        struct Job list = {.prev=NULL, .next=NULL};
```

Comment:
Same for declarations, I think this could be `Job list = ...` now. (But no big deal either way IMO.)

Reply:
_Job's done._